.github/workflows/build.yml

```diff
@@ -122,3 +122,88 @@ jobs:
         run: |
           make docker.buildx.migrate
           make buildx.push.migrate
+
+  unit-tests:
+    name: unit tests 
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
```

Comment:
Do we need to setup golang. Because in [here](https://github.com/openebs/node-disk-manager/blob/master/.github/workflows/pull_request.yml#L45), it works without setting up golang.

Reply:
Ideally it should be but we can use this to control the go version as you suggested :slightly_smiling_face: